src/api/person_operations.py

```diff
@@ -1,13 +1,15 @@
 from connexion.exceptions import ProblemException
 from sqlalchemy import select, text
-from ..database.db_add_table import PersonInDataset
-from ..database.db_operation import get_db_session
+
+from ..database.db_add_tables import PersonInDataset
+from ..database.db_operations import get_db_session
 from datetime import datetime
 from ..config import settings
 from candigv2_logging.logging import CanDIGLogger
 
 logger = CanDIGLogger(__file__)
 
+# --- List persons Endpoint ---
 async def list(dataset_id: int):
     """Lists all person for a given dataset"""
```

Comment:
Comment repetition? The docstring is probably enough

Reply:
Can I keep these comments during the dev? They help me look through my code